src/api/api.js

```diff
@@ -54,6 +54,17 @@ class Api {
       }
 
       if (message.message) {
+        const m = message.message;
+        if (!document.hasFocus()) {
+          new Notification(m.title, {
```

Comment:
showing notification is not a part of API. 
Let's emit an event about it and handle e.g. in `services/notifications.js`